### ISSUE TITLE
[0.11.1] Clustering Documentation not working properly

### ISSUE THREAD

**Issue by vasco-santos** [influxdata/influxdb#6275]
I am running a set of Hybrid Nodes, each one in a Virtual Machine with Ubuntu Server 14.04.

I started by a cluster model similar to the described in the [documentation](https://docs.influxdata.com/influxdb/v0.11/clustering/cluster_setup/). However, following the specified steps didn't work as expected. After some attempts, I accomplished to start correctly the Cluster.

Firstly, when I tried to start influx (CLI), it was not possible because influxd was not previously started (in spite of influxdb service has been previously started).

Afterwards, I started the influx service using the influxd command, using:

`
sudo influxd -config /etc/influxdb/influxdb.conf
`

However, when I tried to start the influxd in the second node, I received errors stating that the node didn't receive answer from the first node (specified in INFLUXD_OPTS).

Therefore, I removed the INFLUXD_OPTS and I started the service of the second node using the following command:

`
sudo influxd -config /etc/influxdb/influxdb.conf -join <IP1>:8091,<IP2>:8091
`

and the third node using the following command:

`
sudo influxd -config /etc/influxdb/influxdb.conf -join <IP1>:8091,<IP2>:8091,<IP3>:8091
`

Finally, when I tried to start the influx CLI, I received a "failed to connect" message. I managed to start the CLI using the following command:

`
influx -host <IP>
`

Summing up, I think that you should review the Clustering Documentation ASAP.

**Comment by e-dard** [comment]
Hi @vasco-santos. Clustering in `0.11` was to be considered experimental, and since we're now on `0.12` I don't think we'll be updating the documentation for `0.11`.

To answer you specific question though, you should start all nodes with all the join args, e.g.,

```
# host 1
$ sudo influxd -config /etc/influxdb/influxdb.conf -join <IP1>:8091,<IP2>:8091,<IP3>:8091
# host 2
$ sudo influxd -config /etc/influxdb/influxdb.conf -join <IP1>:8091,<IP2>:8091,<IP3>:8091
# host 3
$ sudo influxd -config /etc/influxdb/influxdb.conf -join <IP1>:8091,<IP2>:8091,<IP3>:8091
```

> Finally, when I tried to start the influx CLI, I received a "failed to connect" message. I managed to start the CLI using the following command

By default `influx` is going to try and bind to `localhost:8086`. If `influxd` is not running on `localhost` then yes, you'll need to specify the `-host`.

**Comment by aaomoware** [comment]
I believe this is a real issue.

running latest stable influxdb. This "run: flag provided but not defined: -join" is still persistent,

[root@influx-1 influxdb]# sudo influxd -config /etc/influxdb/influxdb.conf -join influx-1:8091,influx-2:8091
flag provided but not defined: -join
usage: run [flags]

run starts the InfluxDB server. If this is the first time running the command
then a new cluster will be initialized unless the -join argument is used.

        -config <path>
                          Set the path to the configuration file.

        -hostname <name>
                          Override the hostname, the 'hostname' configuration
                          option will be overridden.

        -pidfile <path>
                          Write process ID to a file.

        -cpuprofile <path>
                          Write CPU profiling information to a file.

        -memprofile <path>
                          Write memory usage information to a file.

run: flag provided but not defined: -join
[root@influx-1 influxdb]#
